Userland/Libraries/LibJS/Runtime/RegExpPrototype.cpp

```diff
@@ -21,6 +21,8 @@
 
 namespace JS {
 
+static const char* kGroupNames[] = { "", "$1", "$2", "$3", "$4", "$5", "$6", "$7", "$8", "$9" };
```

Comment:
Prefer using `StringView` over a raw string pointer, `snake_case` for variable names, constexpr is free if we're using a `StringView`, and prefer using `Array` over an array, similar to this:

`static constexpr Array<StringView,size> group_names = {...}`

Reply:
Thanks, i'll update later.